module/admin/helper.js

```diff
@@ -146,14 +146,22 @@ module.exports = class AdminHelper {
 	 *
 	 * @param {String} resourceId - ID of the resource to delete.
 	 * @param {String} resourceType - Type of the resource ('program' or 'solution').
+	 * @param {Object} isAPrivateProgram - If Program is Private `true` else `false`.
 	 * @param {String} tenantId - Tenant identifier for multitenancy.
 	 * @param {String} orgId - Organization ID performing the operation.
 	 * @param {String} [deletedBy='SYSTEM'] - User ID or system name that triggered the deletion.
 	 * @param {String} userToken - Auth token used for downstream service calls (e.g., survey service).
 	 *
 	 * @returns {Promise<Object>} - Result object summarizing deletion impact.
 	 */
-	static deletedResourceDetails(resourceId, resourceType, tenantId, orgId, deletedBy = 'SYSTEM') {
+	static deletedResourceDetails(
+		resourceId,
+		resourceType,
+		isAPrivateProgram = 'false',
```

Comment:
@MallanagoudaB isAPrivateProgram = 'false' should not be string why it is in quotes?